src/Helpers.php

```diff
@@ -156,4 +162,118 @@ public static function cleanSchemaName(string $name): string {
 		global $readableAppID;
 		return substr($name, strlen($readableAppID));
 	}
+
+	protected static function getScopeNameFromAttributeArgument(Arg $arg, string $routeName): ?string {
+		if ($arg->name->name === 'scope') {
```

Comment:
The argument could also be unnamed and in that case we'd need to check at which index it is (of the unnamed args).

Reply:
Additionally I would prefer if the check for the right arg is moved out here and this method only handles the arg where we already know that it is the scope arg.

Reply:
> The argument could also be unnamed and in that case we'd need to check at which index it is (of the unnamed args).

I didn't know it's optional, but we could just make it a policy that it's required?

Reply:
> Additionally I would prefer if the check for the right arg is moved out here and this method only handles the arg where we already know that it is the scope arg.

Sure can do that, but then we duplicate it or have another method to verify the name, but might make sense with the position thing as well.

Reply:
You mean by failing if there are unnamed args?

Reply:
yeah